components/organisms/InsightPage/InsightPage.tsx

```diff
@@ -201,16 +194,15 @@ const InsightPage = ({ edit, insight, pageRepos, workspaceId }: InsightPageProps
         body: JSON.stringify({
           name,
           repos: repos.map((repo) => ({ id: repo.id, fullName: repo.full_name })),
-          is_public: isPublic,
+          is_public: true,
```

Comment:
Will these always be public and it's the workspace that determines if the insight is private or not and longterm, we don't need to pass `is_public`?

Reply:
Yes I believe that's what we're going with for now.

Reply:
This will also be handled at the API level: if an insight page is in a public workspace, it's public. If it's in a private workspace, it won't return on the API.

In the future, if we need more granularity, we have those mechanisms already built into the insights stuff.